README.md

```diff
@@ -1,2 +1,114 @@
-# data-notebook-hub
-DATA-3190 Repository of Jupyter Notebooks
+
+# FEWSNET API Notebooks Repository
+
+Welcome to the FEWSNET API Notebooks Repository!
+
+This repository is a curated collection of Jupyter notebooks
+designed to serve as a practical resource for understanding
+and executing public API calls. Our notebooks provide hands-on
+examples, required libs and tutorials that guide you through
+various data queries and manipulations using APIs.
+
+## Features
+
+`Comprehensive Guides:` Step-by-step notebooks on how to
+interact with different public APIs, complete with detailed explanations.
+
+`Ready-to-Use Code:` Snippets that are fully executable and can
+be easily adapted for various data analysis tasks.
+
+`Resource-Rich Documentation:` Markdown cells within notebooks
+offer rich documentation and context for each step of the API interaction process.
+
+`Community-Driven:` Opportunities for the community to contribute and
+enhance the repository with new examples and improved methods.
+
+## Getting Started
+
+To get started:
+
+1. Clone the repository/download the notebook to your local machine.
+2. Create a virtual environment and Install the necessary dependencies
+listed in the requirements.txt file. (Each notebook has a requirements.txt
+file that lists the libraries used).
+3. Open the directory on Jupyter/Jupyterlab
+4. Explore the notebooks directory to find examples relevant to
+your needs and Follow the instructions within each notebook to run the code cells.
+
+## Set-up guide
+
```

Comment:
Let's call this Python Set-up Guide. We may in the future have folks that want to contribute using R, or similar.